booklog/docs/REDIS_INSTALLATION_GUIDE.md

```
@@ -0,0 +1,184 @@
+---
+
+## 🔧 직접 설치 방법 (MSI)
+
+### 1. Redis for Windows 다운로드
+```powershell
+# PowerShell에서 실행 (관리자 권한)
+winget install Redis.Redis
+```
+
+또는 수동 다운로드:
+https://github.com/tporadowski/redis/releases
```

markdownlint-cli2 (0.20.0)

[warning] Bare URL used

(MD034, no-bare-urls)